<launch>
  <arg name="bag" default="/home/yi/Kitti/kitti_2011_09_30_drive_0018_synced.bag"/>
  <arg name="seq" default="4_12"/>

  <rosparam command="load" file="$(find direct_stereo_slam)/cams/kitti/$(arg seq)/T_stereo.yaml" ns="direct_stereo_slam"/>
  
  <node name="direct_stereo_slam" pkg="direct_stereo_slam" type="slam_node" output="screen" required="true">
    <param name="topic0" value="/kitti/camera_gray_left/image_raw" />
    <param name="topic1" value="/kitti/camera_gray_right/image_raw" />
    
    <param name="info0" value="/kitti/camera_gray_left/camera_info" />
    <param name="info1" value="/kitti/camera_gray_right/camera_info" />
    
    <param name="calib0" value="$(find direct_stereo_slam)/cams/kitti/$(arg seq)/camera0.txt" />
    <param name="calib1" value="$(find direct_stereo_slam)/cams/kitti/$(arg seq)/camera1.txt" />
    
    <param name="bag" value="$(arg bag)" />

    <!-- <param name="scale_opt_thres" value="-1" /> -->
    <param name="save" value="true" />
  </node>

  </launch>
